pkg/k8s/apps/translate_test.go

```diff
@@ -1448,3 +1432,477 @@ environment:
 		t.Fatalf("Wrong d generation.\nActual %+v, \nExpected %+v", string(marshalled), string(marshalledOK))
 	}
 }
+
+func Test_translateSfsWithVolumes(t *testing.T) {
+	file, err := ioutil.TempFile("/tmp", "okteto-secret-test")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Remove(file.Name())
+
+	var runAsUser int64 = 100
+	var runAsGroup int64 = 101
+	var fsGroup int64 = 102
+	manifest := []byte(fmt.Sprintf(`name: web
```

Comment:
now that we are on go 1.17, could we use embedded files instead of hardcoding the manifest? (or maybe have a file a simply load it?)